azure-devops/run_build.yml

```diff
@@ -0,0 +1,51 @@
+parameters:
+  targetPlatform: 'x64'
+
+jobs:
+- job: ${{ parameters.targetPlatform }}
+  pool:
+    name: STL
+
+  variables:
+    vcpkgLocation: '$(Build.SourcesDirectory)/vcpkg'
+    vcpkgResponseFile: $(Build.SourcesDirectory)/vcpkg_windows.txt
+  steps:
+    - checkout: self
+      submodules: recursive
+    - task: BatchScript@1
+      displayName: 'Enforce clang-format'
+      condition: eq('${{ parameters.targetPlatform }}', 'x86')
+      inputs:
+        filename: 'azure-devops/enforce-clang-format.cmd'
+        failOnStandardError: true
+      # Install VS 2019 preview on Microsoft Hosted Agents only.
```

Comment:
Remove an indent since this comment applies to the following task? (Or simply remove: this feels like a "what" comment rather than a "why" comment.)